grakn-core/src/main/java/ai/grakn/util/REST.java

```diff
@@ -115,6 +138,11 @@
         public static final String KEYSPACE = "keyspace";
         public static final String FORMAT = "format";
         public static final String UUID_PARAMETER = "uuid";
+        public static final String ATTRIBUTE_TYPE_LABEL_PARAMETER = ":attributeTypeLabel";
```

Comment:
Ahhhh less hard coded strings. I am happy now :)

Reply:
@fppt honestly there's a better way of abstracting these. i'll submit a controller refactor proposal when time permits